sync/src/orphan_block_pool.rs

```diff
@@ -65,6 +88,14 @@ impl OrphanBlockPool {
     pub fn len(&self) -> usize {
         self.parents.len()
     }
+
+    pub fn is_empty(&self) -> bool {
+        self.len() == 0
+    }
+
+    pub fn clone_leaders(&self) -> Vec<ParentHash> {
```

Comment:
Could use impl trait return an Iterator here?

Reply:
Yes, but https://github.com/nervosnetwork/ckb/pull/2906#discussion_r684604252, so I deleted it